src/components/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,79 @@
+import React, {useState} from "react";
+import {Menu, Icon, Button} from "antd";
+import styles from "./Dashboard.module.css";
+
+const Dashboard = props => {
+  const [collapsed, setCollapsed] = useState(false);
+
+  const toggle = () => {
+    setCollapsed(!collapsed);
```

Comment:
I'd expect us to make use of all the antd layout tools here.
Maybe look up and incorporate these:
https://3x.ant.design/components/layout/
https://3x.ant.design/components/grid/